scss/detail.scss

```diff
@@ -0,0 +1,85 @@
+.book-title {
+  height: 39px;
+  width: 210px;
+  color: #000;
+  font-family: Roboto;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+  margin: 10px 0 40px 10px;
+}
+
+.book-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 130px;
+  margin: 0 0 0 10px;
+}
+
+.book-genre {
+  height: 30px;
+  width: 98px;
+  color: #828282;
+  font-family: Roboto;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 28px;
+}
+
+.book-author-title {
+  height: 22px;
+  width: 133px;
+  color: #000;
+  font-family: Roboto;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.book-author-name{
+  height: 22px;
+  width: 274px;
+  color: #828282;
+  font-family: Roboto;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.book-editorial-title {
+  height: 22px;
+  width: 80px;
+  color: #000;
+  font-family: Roboto;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.book-editorial-name {
```

Comment:
Look that some of your class have many similars styles for example .book-editorial-title, book-author-name, book-author-title, book-publish-date-title and  .book-publish-date use these properties

`
  color: #000;
  height: 22px;
  font-family: Roboto;
  font-size: 20px;
  font-weight: 700;
  line-height: 24px;
` 

you can create a general class, something like general text or default-text and in your html use this general class and other class for specify styles.